src/supporting-data/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,280 @@
+      e.target.value.trim().length >= 3
+        ? ''
+        : 'Please enter more details about the database'
+    );
+  };
+
+  return (
+    <SingleColumnLayout>
+      <div className={styles['landing-page']}>
+        <HTMLHead title="Supporting data">
+          <meta name="description" content={metaDescription} />
+        </HTMLHead>
+        <section className="uniprot-grid">
+          <h1 className="uniprot-grid-cell--span-12">Supporting Data</h1>
+          <div
+            className={cn(
+              'uniprot-grid-cell--small-span-12',
+              'uniprot-grid-cell--medium-span-4',
+              styles['image-container']
+            )}
+          >
+            <img
+              src={SupportingDataIllustration}
+              width={300}
+              height={300}
+              alt=""
+            />
+          </div>
+          <div className="uniprot-grid-cell--small-span-12 uniprot-grid-cell--medium-span-8">
+            The Supporting Data section enhances the functionality of UniProt by
+            providing curated and automated datasets that enrich protein entries
+            with detailed information. This section covers keywords, subcellular
+            localization, automatic annotations, functional insights, taxonomy
+            and literature citations. By integrating these data points, it
+            allows researchers to explore proteins more comprehensively,
+            uncovering their roles, interactions, and evolutionary significance.
+            The Supporting Data section serves as a vital resource for making
+            UniProt entries more informative and accessible for biological and
+            biomedical research.
+          </div>
+
+          {/* Categories */}
+          <h2 className="uniprot-grid-cell--span-12">Key Categories</h2>
+          {/* Keywords */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.KeywordsResults],
+                  search: 'query=*',
+                }}
+              >
+                Keywords
+              </Link>
+            </h3>
+            <p>
+              Keywords in UniProt provide standardized terms that summarize the
+              main features and attributes of a protein entry.
+            </p>
+            <p>
+              Categories include: Biological Process, Molecular Function,
+              Technical Terms, Ligands and Cofactors, and Applications.
+            </p>
+          </div>
+          {/* Subcellular Location */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.LocationsResults],
+                  search: 'query=*',
+                }}
+              >
+                Subcellular Location
```

Comment:
```suggestion
                Subcellular Locations
```
Suggesting for consistency with the namespace dropdown in our search bar.